monkestation/code/modules/mob/living/carbon/human/species_type/Tundramoths/tundramoths.dm

```diff
@@ -0,0 +1,73 @@
+/datum/species/moth/tundra
+	name = "\improper Tundra Moth"
+	plural_form = "Tundra Moths"
+	id = SPECIES_TUNDRA
+	mutanteyes = /obj/item/organ/internal/eyes/moth/tundra
+	external_organs = list(/obj/item/organ/external/wings/moth = "Tundra", /obj/item/organ/external/antennae = "Tundra")
+
+	bodypart_overrides = list(
+		BODY_ZONE_HEAD = /obj/item/bodypart/head/tundramoth,
+		BODY_ZONE_CHEST = /obj/item/bodypart/chest/tundramoth,
+		BODY_ZONE_L_ARM = /obj/item/bodypart/arm/left/tundramoth,
+		BODY_ZONE_R_ARM = /obj/item/bodypart/arm/right/tundramoth,
+		BODY_ZONE_L_LEG = /obj/item/bodypart/leg/left/tundramoth,
+		BODY_ZONE_R_LEG = /obj/item/bodypart/leg/right/tundramoth,
+	)
+
+	coldmod = 0.67
+	heatmod = 1.5
+
+
+//Body parts
+/obj/item/bodypart/head/tundramoth
+	icon = 'monkestation/icons/mob/species/tundramoths/bodyparts.dmi'
+	icon_state = "tundra_moth_head"
+	icon_static = 'monkestation/icons/mob/species/tundramoths/bodyparts.dmi'
+	limb_id = SPECIES_TUNDRA
+	is_dimorphic = FALSE
+	should_draw_greyscale = FALSE
+	head_flags = HEAD_LIPS |HEAD_EYESPRITES |HEAD_EYEHOLES | HEAD_DEBRAIN | HEAD_HAIR
```

Comment:
funny trick to make it have the same flags as the normal moth head instead of just copy-pasting them
```suggestion
	head_flags = /obj/item/bodypart/head/moth::head_flags
```